apps/web/src/pages/integrations/UpdateProviderPage.tsx

```diff
@@ -1,42 +1,333 @@
+          handleSubmit((values) => {
+            refetch();
+          });
+          e.preventDefault();
+        }}
+      >
+        <Group spacing={5}>
+          <img
+            src={getLogoFileName(selectedProvider.providerId, colorScheme)}
+            alt={selectedProvider.displayName}
+            style={{
+              height: '24px',
+              maxWidth: '140px',
+            }}
+          />
+          <Controller
+            control={control}
+            name="name"
+            defaultValue=""
+            render={({ field }) => {
+              return (
+                <TextInput
+                  styles={(theme) => ({
+                    root: {
+                      flex: '1 1 auto',
+                    },
+                    wrapper: {
+                      background: 'transparent',
+                      width: '100%',
+                    },
+                    input: {
+                      background: 'transparent',
+                      borderStyle: 'solid',
+                      borderColor: colorScheme === 'dark' ? theme.colors.dark[5] : theme.colors.gray[5],
+                      borderWidth: '1px',
+                      fontSize: '20px',
+                      fontWeight: 'bolder',
+                      padding: 9,
+                      lineHeight: '28px',
+                      minHeight: 'auto',
+                      height: 'auto',
+                      width: '100%',
+                      textOverflow: 'ellipsis',
+                      '&:not(:placeholder-shown)': {
+                        borderStyle: 'none',
+                        padding: 10,
+                      },
+                      '&:hover, &:focus': {
+                        borderStyle: 'solid',
+                        padding: 9,
+                      },
+                      '&:disabled': {
+                        backgroundColor: colorScheme === 'dark' ? colors.B15 : theme.white,
+                        color: colorScheme === 'dark' ? theme.white : theme.black,
+                        opacity: 1,
+                      },
+                    },
+                  })}
+                  {...field}
+                  value={field.value !== undefined ? field.value : selectedProvider.displayName}
+                  // error={showErrors && fieldState.error?.message}
+                  type="text"
+                  data-test-id="title"
+                  placeholder="Enter workflow name"
+                  // disabled={readonly}
+                />
+              );
+            }}
+          />
+          <ActionIcon
+            variant={'transparent'}
+            onClick={() => {
+              navigate('/integrations');
+            }}
+          >
+            <Close color={colors.B40} />
+          </ActionIcon>
+        </Group>
+        <Group mb={16} spacing={16}>
+          <IntegrationChannel name={CHANNEL_TYPE_TO_STRING[selectedProvider.channel]} type={selectedProvider.channel} />
```

Comment:
I believe here it should be <ChannelTitle spacing={5} channel={channelType} /> 
As in the design it is different than how we show it in the list (the IntegrationChannel component)